plugin/manager.go

```diff
@@ -109,22 +109,15 @@ func GetManager() *Manager {
 
 // Init (was NewManager) instantiates the singleton Manager.
 // TODO: revert this to NewManager once we get rid of all the singletons.
-func Init(root, execRoot string, remote libcontainerd.Remote, rs registry.Service, liveRestore bool) (err error) {
+func Init(root string, remote libcontainerd.Remote, rs registry.Service, liveRestore bool) (err error) {
 	if manager != nil {
 		return nil
 	}
 
 	root = filepath.Join(root, "plugins")
-	execRoot = filepath.Join(execRoot, "plugins")
-	for _, dir := range []string{root, execRoot} {
-		if err := os.MkdirAll(dir, 0700); err != nil {
-			return err
-		}
-	}
```

Comment:
should we move the creation of `/run/docker/` (now at [line 128](https://github.com/docker/docker/pull/24646/files#diff-ab0a5ea6d4b42eb51f9e05c2b536a89eR128)) here, to keep the same behavior as before (create the path before constructing the manager) - funny; it appears that previously we created the `runRoot` / `execRoot` twice.

Also looks like  we're no longer creating the `root/plugins` directory now.